wgpu/Cargo.toml

```diff
@@ -47,6 +47,10 @@ fragile-send-sync-non-atomic-wasm = [
 ]
 # Log all API entry points at info instead of trace level.
 api_log_info = ["wgc/api_log_info"]
+# Backends
```

Comment:
would be nice to have all the backend related features in the same place and give them a doc string, i.e. move `angle` and `webgl` down. I want to soon enable proper feature documentation, it's quite messy atm.